components/ngc-float-item-button.component.ts

```diff
@@ -63,12 +69,12 @@ import {
 
   `],
   template: `
-    <div #elementref class="item" (click)="clicked.emit($event)">
+    <div #elementref class="item {{ isDisabled ? 'disabled' : ''}}" (click)="emitClickEvent($event)">
```

Comment:
Okay, the interpolation add the class if isDisabled exist, but the component with isDisabled property if clicked still dispatch the events in the clicked observable.

this is stranger, what you think about?

Reply:
The class `disabled` has `pointer-events` set to `none`, so it really shouldn't pick it up, I've now added an early exit to the emit function, to be sure that nothing will get through.

Reply:
Great!